Sources/Orbit/Support/Components/TimelineItem.swift

```diff
@@ -0,0 +1,302 @@
+import SwiftUI
+
+/// One item of a Timeline.
+///
+/// - Related components
+///   - ``Timeline``
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/progress-indicators/timeline/)
+public struct TimelineItem<Footer: View>: View {
+
+    @Environment(\.sizeCategory) var sizeCategory
+    @Environment(\.horizontalSizeClass) var horisontalSizeClass
+
+    let label: String
+    let sublabel: String
+    let description: String
+    let type: TimelineItemType
+    @ViewBuilder let footer: Footer
+
+    public var body: some View {
+        HStack(alignment: (hasHeaderContent || hasDescription) ? .firstTextBaseline : .top, spacing: .small) {
+
+            TimelineIndicator(type: type)
+                .frame(
```

Comment:
The `frame` and `alignmentGuide` can be encapsulated inside the TimelineIndicator (it seems it partially already is, at least the frame is already inside)

Reply:
Moved inside the component